website/sidebars.js

```diff
@@ -5,63 +5,83 @@ module.exports = {
       label: 'Introduction',
       collapsed: true,
       items: [
-        'intro', 
+        'intro',
       ]
     },
     {
       type: 'category',
-      label: 'General',
+      label: 'Validation',
       collapsed: true,
-      items: [          
-        'allowedrepos',
-        'automount-serviceaccount-token',
-        'block-endpoint-edit-default-role',
-        "block-loadbalancer-services",
-        'block-nodeport-services',
-        'block-wildcard-ingress',
-        'containerlimits',
-        'containerrequests',
-        'containerresourceratios',
-        'containerresources',
-        'disallowanonymous',
-        'disallowedtags',
-        'externalip',
-        'httpsonly',
-        'imagedigests',
-        'noupdateserviceaccount',
-        'poddisruptionbudget',
-        'replicalimits',
-        'requiredannotations',
-        'requiredlabels',
-        'requiredprobes',
-        'storageclass',
-        'uniqueingresshost',
-        'uniqueserviceselector',
-      ],
+      items: [
+        {
+          type: 'category',
+          label: 'General',
+          collapsed: true,
+          items: [
+            'allowedrepos',
+            'automount-serviceaccount-token',
+            'block-endpoint-edit-default-role',
+            "block-loadbalancer-services",
+            'block-nodeport-services',
+            'block-wildcard-ingress',
+            'containerlimits',
+            'containerrequests',
+            'containerresourceratios',
+            'containerresources',
+            'disallowanonymous',
+            'disallowedtags',
+            'externalip',
+            'httpsonly',
+            'imagedigests',
+            'noupdateserviceaccount',
+            'poddisruptionbudget',
+            'replicalimits',
+            'requiredannotations',
+            'requiredlabels',
+            'requiredprobes',
+            'storageclass',
+            'uniqueingresshost',
+            'uniqueserviceselector',
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Pod Security Policy',
+          collapsed: true,
+          items: [
+            'pspintro',
+            'allow-privilege-escalation',
+            'apparmor',
+            'capabilities',
+            'flexvolume-drivers',
+            'forbidden-sysctls',
+            'fsgroup',
+            'host-filesystem',
+            'host-namespaces',
+            'host-network-ports',
+            'privileged-containers',
+            'proc-mount',
+            'read-only-root-filesystem',
+            'seccomp',
+            'selinux',
+            'users',
+            'volumes',
+          ],
+        },
+      ]
     },
     {
       type: 'category',
-      label: 'Pod Security Policy',
+      label: 'Mutation',
```

Comment:
Is there something we can do here to demonstrate that the contents of the "mutation" section are different in-kind from the "validation" section?

Validation is a menu of constraint templates along with example constraints, mutation is merely examples. In other words, "validation" section is like a library, "mutation" is more like a cookbook.

Reply:
I put them in the example section on the website for this reason. Here is the screenshot from the preview -

![image](https://user-images.githubusercontent.com/1626598/215221535-d00e9260-cde7-4343-88a3-64057a2ebd59.png)


But I am open to other ideas.

Reply:
Maybe it's as simple as calling the section "Mutation Examples"? Since the two sections are consumed differently, I'd like to avoid setting up an implicit parallel between the two.

Reply:
+1 on calling it `Mutation Examples`

Reply:
can we describe the intent in the introduction section?

Reply:
Updated template with `Mutation Examples`.

@sozercan do you mean on the main intro page?

Reply:
I mean in https://github.com/open-policy-agent/gatekeeper-library/blob/master/website/docs/intro.md 
which points to 
https://open-policy-agent.github.io/gatekeeper-library/website/